### ISSUE TITLE
Proposal to have output of the data loading functions as pandas DataFrame

### ISSUE THREAD

**Issue by stijnvanhoey** [UGentBiomath/COVID19-Model#89]
@twallema and @JennaVergeynst the current data loading functions are loading a dataframe, but in the function the arrays are extracted as individual numpy arrays again, see https://github.com/UGentBiomath/COVID19-Model/blob/master/src/covid19model/data/sciensano.py#L65-L69

Is there any specific reason to not use the dataframes? I would actually return a Dataframe with the column names the current variable names.

**Comment by twallema** [comment]
It does seem foolish to make this conversion now that you mention it. However, if we just use a pandas data frame, don't we move this type of nastiness:

`np.array([data.loc[:,'TOTAL_IN'].tolist()]).flatten()`

back to the frontend? If I'm honest, I'm only doing this because I have a tendency to cling to Matlab-like matrices and arrays ;). However, I'm guessing your reasons to use pandas are better than my irrational desires to work with Python as if it were Matlab. From my part, you can make the change to pandas. As long as there's some example code to look at, I'll be able to adjust.

**Comment by stijnvanhoey** [comment]
I do understand the transition, been there, done that ;-)

Doing it ot not, I do not think you need the nastiness as you can do the same with Pandas syntax ` data["TOTAL_IN"].values`

I did a quick try to see if these provide equal output
```
>>> data = pd.read_csv("../../data/raw/sciensano/COVID19BE_HOSP.csv")
>>> np.array_equal(np.array([data.loc[:,'TOTAL_IN'].tolist()]).flatten(), data["TOTAL_IN"].values)
True
```
So, I propose: pandas DataFrames instead of the huge tuples as function output (e.g. `index, initial, H_tot, H_tot_cumsum, ICU_tot, H_in, H_out`).

**Comment by twallema** [comment]
Very well, `data["TOTAL_IN"].values` is not nasty at all. I fully agree with the change.

**Comment by stijnvanhoey** [comment]
can be closed, fixed by #90
